PacketDotNet/MiscUtil/IO/EndianBinaryWriter.cs

```diff
@@ -76,7 +76,7 @@ public EndianBinaryWriter (EndianBitConverter bitConverter,    Stream stream, En
         /// </summary>
         public EndianBitConverter BitConverter
         {
-            get { return bitConverter; }
+            get { return this.bitConverter; }
```

Comment:
Why introducing this. everywhere? It's a lot of extra text when its apparent that these are class scoped variables.